### PR TITLE
v4.7.3

### PR DESCRIPTION
**Pull request by Ericgig**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-  number: 1
+  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv  # [not win]
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
+    - cython<=3.0.0                          # [build_platform != target_platform]
```
Comment (Ericgig) [inline recipe/meta.yaml:19]:
```suggestion
    - cython <=3.0.0                         # [build_platform != target_platform]
```

**Comment by Ericgig** [comment]
@conda-forge-admin, please rerender

**Review by hodgestar — APPROVED** [review]
Looks good!

**Comment by hodgestar** [comment]
Ah, the Windows builds are failing with:
```
ERROR: Could not find a version that satisfies the requirement cython<3.0.0,>=0.29.20 (from versions: none)
ERROR: No matching distribution found for cython<3.0.0,>=0.29.20
```
Maybe Cython didn't build a 0.29.20 for Windows and we have to relax the requirement a bit?

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
-    - cython                                 # [build_platform != target_platform]
+    - cython <=3.0.0                         # [build_platform != target_platform]
     - numpy                                  # [build_platform != target_platform]
     - {{ compiler('cxx') }}
   host:
     - python
     - pip
     - numpy {{ numpy }}
-    - cython >=0.29.20
+    - cython >=0.29.20,<=3.0.0
```
Comment (Ericgig) [inline recipe/meta.yaml:26]:
```suggestion
    - cython <=3.0.0
```

**Pull request merged**